tests/apollo/util/bft.py

```diff
@@ -133,6 +133,8 @@ async def wrapper(*args, **kwargs):
                 bft_network = kwargs.pop("bft_network")
                 bft_network.is_existing = True
                 with log.start_task(action_type=async_fn.__name__):
+                    if rotate_keys:
+                        await bft_network.do_key_exchange()
```

Comment:
If the `bft_network` is externally provided (such as in Hermes), calling `do_key_exchange()` here probably doesn't make sense...